release/long_running_tests/workloads/many_logs.py

```diff
@@ -0,0 +1,45 @@
+import ray
+import argparse
+
+import random
+import string
+import logging
+
+parser = argparse.ArgumentParser(prog="Stress Test - Logging")
+parser.add_argument(
+    "--total-num-tasks",
+    type=int,
+    help="Total number of tasks sending logs",
+    required=True,
+)
+parser.add_argument(
+    "--log-lines-per-task",
+    type=int,
+    help="Total lines of logs to be sent",
+    required=True,
+)
+
+parser.add_argument("--log-line-size", type=int, help="Size of a log", required=True)
+
+args = parser.parse_args()
+ray.init(address="auto")
+
+logger = logging.getLogger(__name__)
+
+
+@ray.remote
+def gen_logs(log_size, log_num):
+
+    letters = string.ascii_uppercase + string.digits
+
+    for _ in range(log_num):
```

Comment:
Instead of making this count based, just log 100KB lines as fast as possible for 5 minutes, or something like that.